docs/learn/software_overview.md

```diff
@@ -1,53 +1,54 @@
-# Software Overview
+# Workflow & Software Overview
 
-Starting out in the CNC world can be very intimidating, even if you already know about 3D printing
-(which is a CNC machine, technically). Here, we will break down the different software components
-and tools, to give you a better understanding of some of the terminology, which gives you the
-language to find help. Also, we want you to have an overview of the area, so you can understand all
-of the parts, and how they work together.
+Starting out in the [CNC](https://en.wikipedia.org/wiki/Numerical_control) world can be very intimidating, even if you already know about 3D printing (which is a CNC machine, technically).  However, when broken down into manageable pieces it's easy to digest.  In the world of CNC, workflow and software are intrinsically linked. 
+
+Here, we will describe the different steps in the CNC milling workflow process along with their associated software components/tools.  The goal is to provide you a better understanding of some of the steps and terminology, which in turn gives you the background to better seek help and explore possibilities.
 
 ## Overview
 
-This is the 1000ft (304m) overview.
+This is the 1000ft (304m) overview (we will explore each in more detail within the sections that follow).
+
+There are 4 components to the workflow and associated software as follows:
 
-There are 4 parts: CAD, CAM, Sender, Controller
+| |Step|Software|
+|-|----|--------|
+|1|Design|CAD (Computer Assisted Design) and/or VGE (Vector Graphics Editor)|
+|2|Toolpathing|CAM (Computer Assisted Machining)|
+|3|G-Code Sending|Sender|
+|4|Milling|Controller/Firmware|
 
 TODO Insert a diagram showing the workflow.
 
-[CAD: Computer Assisted Design](#cad-computer-assisted-design)
-:   Before you do anything, you have to take your idea and get it into the computer. This can take
-    many forms, but the output is either a 2D file (like a .DXF or .SVG) or sometimes (but not
-    often) a 3D file (like a .STL).
+[Design/CAD/Vector Graphics Editor](#design-cad-vge)
+:   For the most part, you are either going to use your CNC machine to create a part (e.g. a wooden or metal gear) or for something         artistic (e.g. a sign or carving). If the circumstance is that you are needing to create the design from scratch you will likely be       using CAD or Vector Graphics Editor software to do so.  It is worth noting that this is NOT a certainty.  Many people may already       have these designs in hand provided from some other source.  Regardless, the output of this step (and input to the next) is             typically either a 2D file (like a .DXF or .SVG) or sometimes (but not often) a 3D file (like a .STL).
```

Comment:
Many of the steps could be skipped. I don't think it's necessary to mention that you don't need a design. Maybe I'm just being sensitive.